src/backend/distributed/utils/namespace_utils.c

```diff
@@ -0,0 +1,35 @@
+/*-------------------------------------------------------------------------
+ *
+ * namespace_utils.c
+ *
+ * Utility functions related to namespace changes.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "catalog/namespace.h"
+#include "distributed/namespace_utils.h"
+
+/*
+ * PushAndNullifySearchPath pushes search_path to be NIL and sets addCatalog to
```

Comment:
I think you can simplify this comment since some details are mentioned below

Reply:
I purpose removing the comment within the function totally